osbs/tekton.py

```diff
@@ -506,22 +506,36 @@ def get_error_message(self):
                 err_message += f"{plugin} : {error}\n"
 
         err_message += "\npipeline run errors:\n"
-        task_runs_status = data['status']['taskRuns']
 
-        for task_name, stats in task_runs_status.items():
-            if stats['status']['conditions'][0]['reason'] == 'Succeeded':
-                continue
-
-            err_message += f"pipeline task '{task_name}' failed:\n"
+        if 'taskRuns' in data['status']:
+            task_runs_status = data['status']['taskRuns']
```

Comment:
task_runs_status = data['status'].get('taskRuns') or something similiar would work the same and we could save one conditional statement

Reply:
and one indent level

Reply:
I am sorry, I meant this
`task_runs_status = data['status'].get('taskRuns', {})`

Reply:
ok will do that, but won't save conditional anyway since I have to do else part

Reply:
ah, then you can keep the original condition if you want.

Reply:
already changed that